modules/network/arbitrum/index.ts

```diff
@@ -0,0 +1,43 @@
+import { JsonRpcProvider } from '@ethersproject/providers';
+import { data } from './data';
+import { workerJobs } from './workers';
+import { createServices } from './services';
+
+let services: ReturnType<typeof createServices>;
+
+export class arbitrumNetworkConfig {
```

Comment:
Can you capitalize the first letter now that these are classes

Reply:
done